lib/pbench/server/api/resources/pbench_users.py

```diff
@@ -265,29 +356,40 @@ def get(self, username):
         """
         # get the auth token
         auth_header = request.headers.get("Authorization")
-        auth_token = ""
-        if auth_header:
-            try:
-                auth_token = auth_header.split(" ")[1]
-            except IndexError:
-                self.app.logger.info(f"Bearer token malformed: {auth_header}")
-                abort(401, message="Bearer token malformed")
 
-        if auth_token:
-            user_id = UserModel.decode_auth_token(auth_token, self.app)
-            user = UserModel.query.filter_by(username=username).first()
-            if user.id != user_id:
-                self.app.logger.info(
-                    f"User_id decoded from the auth token {user_id} is different for the user's id {user.id}"
-                )
-                abort(403, message="Authentication token does not belong to the user")
+        if not auth_header:
+            self.app.logger.error(f"Auth token not provided during logout")
+            abort(
+                400,
+                message="Please add 'Authorization' token as Authorization: Bearer <JWT_Auth_token>",
+            )
+
+        try:
+            auth_token = auth_header.split(" ")[1]
```

Comment:
If we're going to standardize on `Authorization: Bearer <token>` then we should check that `split()[1]`.lower() is "bearer". (Or is it supposed to be case sensitive? I lowercased it in my check!)

Reply:
ahh yeas